### PR TITLE
Change Gazebo SRDF to be a xacro for parser coverage purposes

### PR DESCRIPTION
**Pull request by sea-bass**
Since we're adding this feature to MoveIt Studio (hint hint @zultron), I wanted to change one of our configuration packages to use a xacro SRDF to ensure we're testing it regularly.

### CODE REVIEW

File: src/picknik_ur_gazebo_config/config/moveit/picknik_ur_gazebo.srdf.xacro [hunk 1]
```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--This does not replace URDF, and is not an extension of URDF.
+    This is a format for representing semantic information about the robot structure.
+    A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
+-->
+<robot name="ur5e" xmlns:xacro="http://wiki.ros.org/xacro">
+    <!--Xacro Parameters-->
+    <xacro:arg name="gripper_name" default="$(arg robotiq_85)"/>
```
Comment (MarqRazz) [inline src/picknik_ur_gazebo_config/config/moveit/picknik_ur_gazebo.srdf.xacro:8]: I don't understand why the default is an additional arg. I expected it to just be 
`<xacro:arg name="gripper_name" default="robotiq_85"/>`

Comment (sea-bass) [reply src/picknik_ur_gazebo_config/config/moveit/picknik_ur_gazebo.srdf.xacro:8]: oh yeah that is search + replace carelessness. Good catch

Comment (sea-bass) [inline src/picknik_ur_gazebo_config/config/moveit/picknik_ur_gazebo.srdf.xacro:8]:
```suggestion
    <xacro:arg name="gripper_name" default="robotiq_85"/>

**Review by MarqRazz — APPROVED** [review]
Everything here looks good. I tried to test it out but switching between `main` on Studio and the xacro branch just blows up my workspace and submodules.

**Pull request merged**